mlir/include/mlir/Dialect/MIOpenOps/LowerMIOpenOps.h

```diff
@@ -86,6 +86,10 @@ struct Conv2DRewritePattern : public OpRewritePattern<T> {
     // - Part 2: PassThrough non-K dimension to dimension 1, name it as gemmM.
```

Comment:
I'm sure you have all the changes ready and tested, but the diff is almost impossible to review because:

- Seems that you included the on-going lowering side.
- You started from a commit that is back to history, but the merging ended up making it `look like` new changes.
- Formatting changes are in mixed manner with non-formatting changes.

I have a few suggestions going forward:

- Use rebase instead of merge, since merge will generate unnecessary commits
- Make sure the formatting changes happens in a separate commit, not everything al-together. This way the reviewer can go through your diffs in non-formatting changes

Regarding on this review, I think it makes sense to do the following:

1. Start from a new branch
2. Do a clang-format on files you want to update formatting, make the 1st commit
3. Check-out non-lowering files from the current branch, make the 2nd commit

Though the process, I can do my review in looking at what changed in the 2nd commit.

Reply:
BTW there's a similar reminder that you will need to update and include the tests related translation in `mlir/test/Dialect/MIOpen`